examples/jenkins/jenkins-persistent-template.json

```diff
@@ -113,6 +113,14 @@
                 },
                 "env": [
                   {
+                    "name": "OPENSHIFT_ENABLE_OAUTH",
+                    "value": "anyvalue"
+                  },
+                  {
+                    "name": "OPENSHIFT_ENABLE_REDIRECT_PROMPT",
+                    "value": "anyvalue"
+                  },
+                  {
                     "name": "JENKINS_PASSWORD",
                     "value": "${JENKINS_PASSWORD}"
```

Comment:
you are correct ... i keep muddling `oc new-app -e JENKINS_PASSWORD ... <image ref>` with `oc new-app jenkins-ephemeral` .... removing JENKINS_PASSWORD from template.  Updating template description.